Libs/Widgets/ctkSettingsDialog.cpp

```diff
@@ -167,11 +167,11 @@ void ctkSettingsDialogPrivate::updateRestartRequiredLabel()
   bool restartRequired = !restartRequiredSettings.isEmpty();
   if (restartRequired)
     {
-    QString header = q->tr(
+    QString header = ctkSettingsDialog::tr(
```

Comment:
If not too complicated, remove HTML tags from translatable text.

Reply:
Sure, it won't be a problem.
I planed to do it in the upcoming PR related to string marking in DICOM lib. In this PR, I wanted to only correct the code so that lupdate warnings disappear.

Please let me know if it's better to do it there.

Reply:
Thanks, HTML tags are now separated from the translatable text.